packages/ra-core/src/auth/useLogoutIfAccessDenied.spec.tsx

```diff
@@ -79,56 +76,67 @@ describe('useLogoutIfAccessDenied', () => {
     });
 
     it('should not logout if passed no error', async () => {
-        const { queryByText } = renderInRouter(
-            <AuthContext.Provider value={authProvider}>
-                <TestComponent />
-            </AuthContext.Provider>
-        );
+        render(<Route path="/" element={<TestComponent />} />, {
+            wrapper: TestWrapper,
+        });
 
         await waitFor(() => {
             expect(authProvider.logout).toHaveBeenCalledTimes(0);
```

Comment:
In fact, we should check that checkauth has been called before

Reply:
No this test is actually useless as it tests that the `TestComponent` does not call `logoutIfAccessDenied` if not passed an error